src/components/map/MapConfig.ts

```diff
@@ -1,8 +1,9 @@
-import {CameraStop, MapViewProps} from '@rnmapbox/maps';
+//import {CameraStop, MapViewProps} from '@rnmapbox/maps';
```

Comment:
If you have good reasons for removing the type for `MapViewConfig` you should remove his commented out import as well

Reply:
Thanks I will fix it good catch ;), I haven't properly double-checked it just a small draft to let @marius-at-atb test his changes.